src/environment.cc

```diff
@@ -105,6 +105,13 @@ std::string Environment::GetMetadataString(const std::string& path) const {
   }
 }
 
+bool Environment::IsGcpLocationZonal(const std::string& location) const {
+  // Use the number of dashes to distinguish between zones (e.g.
+  // "us-central1-a", "us-east1-b") and regions (e.g."us-central1", "us-east1").
```

Comment:
Mention "global" here too?

Reply:
In the current use cases, we are only distinguishing between regional and zonal locations. Perhaps drop the test for "global"?

Reply:
Sure, either way. This is just so that future users of this function know what the meaningful inputs are.

Reply:
done - removed the test for "global"